src/multiple_value/multiple_value.js

```diff
@@ -166,8 +168,8 @@ class MultipleValue extends React.PureComponent {
                     color={config[`style_${dataPoint.name}`]}
                     onClick={() => { this.handleClick(dataPoint, event) }}
                     layout={this.getLayout()}
-                  >
-                    {dataPoint.formattedValue}
+                  >                  
+                  { dataPoint.html ? ReactHtmlParser(dataPoint.html) : dataPoint.formattedValue } 
```

Comment:
Hm, a little scary!

See security section - https://github.com/peternewnham/react-html-parser#readme

"All HTML should be properly sanitized using a dedicated sanitisation library"

Reply:
Ahh. Good catch. Thanks. Added an sanitizer.